### ISSUE TITLE
[Suggestion] Open up mod quests earlier

### ISSUE THREAD

**Issue by jolandepoel** [Alnarra/WaySouls#4]
I was playing through your pack and I couldn't figure out why I couldn't start any magic mod quests. In my experience, you need very little to start in Thaumcraft or Botania or any of the others. Turns out (after going into bq_admin edit mode just to be able to see this) you need a series of quests in the second category: a diamond pickaxe, obsidian, kill a blaze, kill an enderman. 

First of all: don't hide those mainline quests that gate other quests, it makes it very hard to see which line to follow if you just want to get into a mod. 

Second of all: Why? No really, why?! You can make a resonating wand with an ender pearl, which you can get from an enderman way before you even visit the Nether and have to traverse a hostile environment just to find a fortress and a blaze. Or, if you're lucky, it's simply in one of the Astral Sorcery chests at the natural generating altars. I had an ender pearl before I even saw an enderman. And gating ender pearls behind blaze rods is really counter intuitive. 

For Thaumcraft, you need gold to make the Thaumometer, but that's it. You don't need a diamond pick for that. Or any of the other stuff! I would say: as soon as players made the book: open up the mod category. I was so disappointed when I successfully made the Thaumcraft book, and still couldn't do any Thaumcraft quests. 

The other early game mods are in a similar or even worse boat, since they can be played with resources you can just find lying around. Not need to kill any blazes or enderman. 

As a side note, the Better Questing Mob Hunt task is notorious for not working/not detecting correctly, I would suggest replace 'kill blaze' and 'kill enderman' with 'get blaze rod' and 'get ender pearl'.

**Comment by Alnarra** [comment]
I believe those had been corrected in the beta builds I was working on in December. Thaumcraft as of yet didn't have any quest (in large part because I was trying to avoid stepping on the toes of the Thaumcraft book itself). 

Haven't had a lot of time to work on things, have been busy IRL with work but may get some time freeded up hopefully in June to at least get what changes I had made back in Dec published.
